config/cli.go

```diff
@@ -100,17 +100,22 @@ func addCmd(c *cli.Context) error {
 	if c.NArg() > 1 {
 		return cliutils.PrintHelpAndReturnError("Wrong number of arguments.", c)
 	}
-	return addOrEdit(c, false)
+	if c.Bool("overwrite") {
+		return addOrEdit(c, nil)
+	}
+	expectExists := false
+	return addOrEdit(c, &expectExists)
 }
 
 func editCmd(c *cli.Context) error {
 	if c.NArg() != 1 {
 		return cliutils.PrintHelpAndReturnError("Wrong number of arguments.", c)
 	}
-	return addOrEdit(c, true)
+	expectExists := true
+	return addOrEdit(c, &expectExists)
 }
 
-func addOrEdit(c *cli.Context, expectExists bool) error {
+func addOrEdit(c *cli.Context, expectExists *bool) error {
```

Comment:
I think it would be clearer if we had an enum instead of a pointer to boolean. For example:
```go
type configOp int

const (
	configAdd configOp = iota
	configEdit
	configOverwrite
)
```

```suggestion
func addOrEdit(c *cli.Context, configOp configOp) error {
```